src/LtiServiceConnector.php

```diff
@@ -172,6 +194,32 @@ public function getAll(
         return $results;
     }
 
+    private function logRequest(
+        int $requestType,
+        IServiceRequest $request,
+        array $responseHeaders,
+        ?array $responseBody
+    ): void {
+        $contextArray = [
+            'request_method' => $request->getMethod(),
+            'request_url' => $request->getUrl(),
+            'response_headers' => $responseHeaders,
+            'response_body' => json_encode($responseBody),
+        ];
+
+        $requestBody = $request->getPayload()['body'] ?? '';
+
+        if (!empty($requestBody)) {
+            $contextArray['request_body'] = $requestBody;
+        }
```

Comment:
I think this can be just `$contextArray['request_body'] = $requestBody ??= '';`

Reply:
Mmm, I thought it'd make more sense not to have `request_body` present in the logs if there isn't one (like if it's a `GET` request) - I don't feel strongly either way though, do you still think it should be added?